alpaca_trade_api/rest.py

```diff
@@ -304,11 +306,14 @@ def submit_order(self,
         """Request a new order"""
         params = {
             'symbol':        symbol,
-            'qty':           qty,
             'side':          side,
             'type':          type,
             'time_in_force': time_in_force
         }
+        if qty is not None:
+            params['qty'] = qty
+        if notional is not None:
+            params['notional'] = notional
```

Comment:
Add validation to make sure only one is set ? I see other parameters don't have any validation here but since this is a new breaking change, it would be good to catch early.

Reply:
this validation is ok, but not mandatory. if the user mis-use this api - the server will let them know. 
think about it - this mistake happens once BUT this validation happens every time the user do submit_order.
I know we are not a real time api, but this does add latency (every time).

Reply:
The practice of this package has been a simple wrapper of the API and to rely on the API for validation unless it's the language specific checks. I would leave it to the API side to check to avoid unnecessary inconsistency between checks in the client side and API side.

Reply:
Thanks for the context here! I'll strip out the validation and leave it to the API.